client.go

```diff
@@ -215,6 +215,12 @@ func populateClientConfig(config *Config, createdPacketConn bool) *Config {
 	if config.IdleTimeout != 0 {
 		idleTimeout = config.IdleTimeout
 	}
+	attackTimeout := protocol.DefaultAttackTimeout
+	if config.AttackTimeout > 0 {
+		attackTimeout = config.AttackTimeout
+	} else if config.AttackTimeout < 0 {
+		attackTimeout = 0
```

Comment:
You're right - it's a little odd. I did this so that a 0 value for AttackTimeout would indicate the default, which could be non-zero, and a negative value would indicate 0. But maybe this is fragile and could be improved by a mitigationOn flag in the config.

Reply:
That makes sense; it just needs to be documented.  I think a comment explaining the intended semantics would be enough.

Reply:
@marten-seemann what's your take on this? Should it be a bool vs a timeout with negative values?